experimenter/experimenter/experiments/tests/test_change_formatting_utils.py

```diff
@@ -284,6 +284,16 @@ def test_formatting_for_array_fields(self):
             experiment, self.user, "primary outcomes were added", timestamp_2
         )
 
+        old_value_segments = []
+        new_value_segments = ["test-segment"]
```

Comment:
I can't comment on outcome lines, but can you update the outcome vars on lines 277-278 to make them more clear now that there are segments in here too? `old_value` --> `old_value_outcomes` for example

Reply:
And same for the other vars that are currently generic but should be distinguished from the segments vars.